### PR TITLE
Setting variables on init db script

### PR DESCRIPTION
**Pull request by sgrazi**
## Description:
Changed init.sh to use environment variables for the user and password. Updated docker-compose images and documentation alongside this change

## Related Issue (if appropriate):
https://app.asana.com/0/1205530699145161/1206048607296779/f

## Steps to test change:
- Tear down your video-headline environment
`docker-compose down -v   # This will stop and remove containers and volumes`
- Build it again
`docker-compose up --build  # This will rebuild and start the containers`
- VideoHeadline should be functional

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+runme:
+  id: 01HGE0ZDMVJFAC0WHX7TBH7T9C
+  version: v2.0
+---
+
```
Comment (lucilamoreno) [inline CONTRIBUTING.md]: can we remove this?

Comment (sgrazi) [reply CONTRIBUTING.md]: done, thanks for pointing out

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -128,15 +131,16 @@ Follow the steps below to set up ESLint for linting React code in Visual Studio
...
-  "editor.formatOnSave": true,
-  "[javascriptreact]": {
-  	"editor.defaultFormatter": "rvest.vs-code-prettier-eslint"
-    },
-    "[json]": {
-  	"editor.defaultFormatter": "rvest.vs-code-prettier-eslint"
-  }
-  ```
+
+```json {"id":"01HGE0ZDMTZ1QSB2P8MF8PGX1Y"}
```
Comment (lucilamoreno) [inline CONTRIBUTING.md]: Is this `{"id":"01HGE0ZDMTZ1QSB2P8MF8PGX1Y"}` okey?

**Review by lucilamoreno — APPROVED** [review]
Left a comment. Otherwise, it looks good

File: CONTRIBUTING.md (hunk 2 above)
Comment (sgrazi) [reply CONTRIBUTING.md]: disabled extension so this wont happen again X)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -55,8 +59,7 @@ Create a .env file at the root of the project with all the variables defined in
 
 ##### .env variables
 
-1. `DATABASE_HOST`, `DATABASE_PORT`, `DATABASE_USER`, `DATABASE_PASSWORD`: Database credentials for the PostgreSQL database.
-   **Note:** If you need to change the database credentials, you need to modify the values in the .env file and also the line 13 in the docker-commands/db_init/init.sh file (`CREATE USER qualabs WITH PASSWORD 'yVm=7>GQ';`) with your new values, since the file does not use the .env variables.
```
Comment (NoeliaBentancor) [inline CONTRIBUTING.md:59]: This is not necessary anymore?

**Comment by NoeliaBentancor** [comment]
We should also update .env.example with new env variables

**Comment by lucilamoreno** [comment]
> We should also update .env.example with new env variables

@NoeliaBentancor I think this variables were already in the .env-example

**Comment by sgrazi** [comment]
> We should also update .env.example with new env variables

yup, as Luli said, those are already exisiting variables

**Review by NoeliaBentancor — APPROVED** [review]

**Pull request merged**